### PR TITLE
Fixed support for blacklisting PR comments

### PR DESCRIPTION
**Pull request by avyayv**
See issue #471

### CODE REVIEW

**Comment by avyayv** [comment]
The way to update in config file:
`blacklist_pull_request_users = ['coveralls']`

File: apertium_wikistats_bot.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 apiURL = 'http://wiki.apertium.org/w/api.php'
```
Comment (sushain97) [inline apertium_wikistats_bot.py]: All the changes to this file seem irrelevant?

**Comment by avyayv** [comment]
Should work now.

File: modules/git.py [hunk 2]
```diff
@@ -316,15 +316,16 @@ def do_POST_unsafe(self, data):
...
-
                 if action == 'deleted':
                     messages.append(template.format(repo, user, number, url))
                 else:
                     template = '{:}: {:} * review comment {:} on pull request #{:}: {:} {:}'
-                    messages.append(truncate(
-                        data['comment']['body'],
-                        template.format(repo, user, action, number, '{}', url)
-                    ))
+                    blacklist_pull_request_comment_users = config.blacklist_pull_request_comment_users
```
Comment (sushain97) [inline modules/git.py]: Can we make this default to the empty set?

File: modules/git.py [hunk 3]
```diff
@@ -316,15 +316,16 @@ def do_POST_unsafe(self, data):
                 number = data['pull_request']['number']
                 url = data['comment']['html_url']
                 action = data['action']
-
```
Comment (sushain97) [inline modules/git.py]: Any reason for this change?

File: modules/git.py (hunk 2 above)
Comment (scoopgracie) [reply modules/git.py]: Fixed

File: modules/git.py (hunk 3 above)
Comment (scoopgracie) [reply modules/git.py]: PEP8. There is not a big change in what is happening at line 319.
